types/create-cert/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for create-cert 1.0
+// Project: https://github.com/lukechilds/create-cert
+// Definitions by: Chris Midgley <https://github.com/midgleyc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.4
```

Comment:
```suggestion
```
I'd argue you no longer need this. It's not used during the CI already
and you're not referencing anything directly

Reply:
Sorry, missed this comment, only saw the tick below.

When I ran `npm test create-cert` from the root of the directory, it failed with this error -- I had to add a minimum TypeScript version to get it to run at all, and then it raised some errors for me to fix. Does the CI run something else?

Reply:
runs same, but maybe you're local TS repo was outdated. I've run PR locally, no problems